Bluepill-runner/Bluepill-runner/BPReportCollector.h

```diff
@@ -17,6 +17,7 @@
  * @param finalReportPath the path to save the final report
  */
 + (void)collectReportsFromPath:(NSString *)reportsPath
+                   applyXQuery:(NSString *)XQuery
```

Comment:
Another nit: `andApplyXQuery` and move it after `onReportsCollected`. I think the last arg should've been `withOutputAtPath` to follow Objective-C convention so feel free to fix that as well ;-)

Reply:
Fixed.